packages/flutter/lib/src/material/selectable_text.dart

```diff
@@ -690,7 +705,9 @@ class _SelectableTextState extends State<SelectableText>
         cursorColor =
             widget.cursorColor ?? selectionStyle.cursorColor ?? cupertinoTheme.primaryColor;
         selectionColor =
```

Comment:
Instead of updating each switch case, your `selectionColor: widget.selectionColor ?? selectionColor` on `EditableText` is sufficient.

Reply:
I adjusted it thank you! Does that mean cursorColor could also be shortened here? Or am I missing something

Reply:
I think for this PR we can leave it as is.